.github/workflows/win-wsl.yaml

```diff
@@ -0,0 +1,314 @@
+on:
+
+  workflow_call:
+
+jobs:
+  smoketest:
+    name: Windows Smoketest
+    runs-on: windows-2022
+
+    steps:
+      - name: "Build :: Checkout"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # for `git describe`
+          persist-credentials: false
+      - name: Pre-create HNS networking
+        run: |
+          $ErrorActionPreference = "Stop"
+          $PSNativeCommandUseErrorActionPreference = $true
+          Write-Host "Disabling Windows Firewall"
+          & Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
+
+          # Download the MS helpers and load them
+          $u = "https://raw.githubusercontent.com/Microsoft/SDN/master/Kubernetes/windows/hns.psm1"
```

Comment:
Can we refactor the "master" part to a version number, or at least to a fixed commit? We could let Renovate Bot update this, then.